.github/workflows/announce-lts-rc.yml

```
@@ -8,9 +8,23 @@ jobs:
   post:
     runs-on: ubuntu-latest
     steps:
-      - uses: roots/discourse-topic-github-release-action@main
+      - name: Post on Discourse
+        uses: roots/discourse-topic-github-release-action@main
```

```suggestion
        uses: roots/discourse-topic-github-release-action@fc9e50fa1a1ce6255ba4d03f104382845b79ad5f # v1.0.0
```
(nit) pinning this GHA version to avoid potential bad surprises.